### DISCUSSION TITLE
500 server error on /setup/

### DISCUSSION THREAD

**Discussion by dietrichmd** [TandoorRecipes/recipes#308] [Q&A]
Got everything installed with a few hiccups getting apache and gunicorn to play nicely.  That said, when I try to load recipes, it tries to redirect me to /setup/ which gives me a 500 server error.

![image](https://user-images.githubusercontent.com/30147271/103469307-b83d7c80-4d28-11eb-9454-ea4b4aa45523.png)

Any ideas?

Thanks

**Comment by aarondoet** [comment]
Idk if that is actually the problem but try doing what the red text says

**Reply by dietrichmd** [reply]
yeah.  ive run the migration and each time get no errors just that it completed.  then i run the app and get that.  so not quite sure which is telling me what.  

![image](https://user-images.githubusercontent.com/30147271/103469910-b972a780-4d30-11eb-8b39-5a4849546db8.png)

**Reply by aarondoet** [reply]
Maybe bc you are using sudo on it (at least in this screenshot)? You are already root so that shouldn't even make a difference if you use it or not but who knows. If that is not the problem then idk.

**Reply by dietrichmd** [reply]
surprisingly, if i dont run it as sudo, i get weird errors with the script itself about elevated permissions.

![image](https://user-images.githubusercontent.com/30147271/103469946-2d14b480-4d31-11eb-9730-74110ac48d9e.png)

**Reply by vabene1111** [reply]
You are running a manual setup and it looks like your django does not have the appropriate permissions to connect to the database thus running the migrations fails which in turn breaks everything else
